recipes/libfabric/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class LibfabricConan(ConanFile):
+    name = "libfabric"
+    description = "Open Fabric Interfaces"
+    topics = ("conan", "fabric")
```

Comment:
```suggestion
    topics = ("fabric", "communication", "framework", "service")
```

Try to use something closer to the project tags